src/components/shared/misc/SchedulerEditor.vue

```diff
@@ -0,0 +1,405 @@
+                                <label>November</label>
+                            </div>
+                        </div>
+
+                        <div class="field">
+                            <div class="ui toggle checkbox">
+                                <input type="checkbox" value="1" v-model="schedule.months.december">
+                                <label>December</label>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="column">
+                <div class="ui form">
+                    <div class="grouped fields">
+                        <div class="field">
+                            <label>Starts at</label>
+                            <croud-datepicker ref="startDate" placeholder="Select start date.." :date="schedule.limit.from" v-on:date-selected="setStartDate"></croud-datepicker>
+                        </div>
+
+                        <div class="field">
+                            <label>Ends at</label>
+                            <croud-datepicker ref="endDate" placeholder="Select end date.." :date="schedule.limit.to" :minDate="getStartDate" v-on:date-selected="setEndDate"></croud-datepicker>
+                        </div>
+                        <button class="ui mini button" @click="clearEndDate">Clear End Date</button>
+                    </div>
+
+                    <div class="field">
+                        <label>Time this schedule runs at</label>
+                        <semantic-form-dropdown v-model="schedule.at" :options="timeOptions"></semantic-form-dropdown>
+                        <div class="ui light" style="margin-top:5px">
+                            (Timezone: <strong>{{ schedule.timezone }}</strong>)
+                        </div>
+                    </div>
+
+                    <div class="field">
+                        <label>Maximum number of times to run</label>
+                        <input type="number" v-model="schedule.limit.count">
+                    </div>
+
+                    <div class="ui divider"></div>
+
+                    <div class="field">
+                        <button @click="done" class="ui blue fluid button">
+                            Done
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import { defaultsDeep } from 'lodash'
+    import moment from 'moment'
+
+    import CroudDatepicker from '../forms/DatePicker'
+
+    /**
+    * Our scheduler editor
+    *
+    * @example ./croud-scheduler-editor.md
+    */
+
+    export default {
+        name: 'croud-scheduler-editor',
+
+        model: {
+            prop: 'schedule',
+        },
+
+        props: {
+            /**
+             * Scheduler Schema
+             *
+             * @ignore
```

Comment:
Why are we excluding this prop from the docs?

Reply:
As we are instructing users to pass the schedule object through the v-model, my thought process was to hide this prop as it may be confusing?

Reply:
You didn't do that with the summary dropdown component in the last PR? I think we usually add 'v-model alias' to the doc block